patches/api/0480-Expose-server-build-information.patch

```diff
@@ -0,0 +1,260 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: masmc05 <masmc05@gmail.com>
+Date: Tue, 14 May 2024 21:38:49 -0700
+Subject: [PATCH] Expose server build information
+
+Co-authored-by: Riley Park <rileysebastianpark@gmail.com>
+
+diff --git a/src/main/java/io/papermc/paper/ServerBuildInfo.java b/src/main/java/io/papermc/paper/ServerBuildInfo.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..cbd7aba225c91015684a561278b8b42e685f35ff
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/ServerBuildInfo.java
+@@ -0,0 +1,121 @@
++package io.papermc.paper;
++
++import java.time.Instant;
++import java.util.Optional;
++import java.util.OptionalInt;
++import net.kyori.adventure.key.Key;
++import net.kyori.adventure.util.Services;
++import org.jetbrains.annotations.ApiStatus;
++import org.jetbrains.annotations.NotNull;
++
++/**
++ * Information about the current server build.
++ */
++@ApiStatus.NonExtendable
++public interface ServerBuildInfo {
++    /**
++     * The brand id for Paper.
++     */
++    Key BRAND_PAPER_ID = Key.key("papermc", "paper");
++
++    /**
++     * Gets the {@code ServerBuildInfo}.
++     *
++     * @return the {@code ServerBuildInfo}
++     */
++    static @NotNull ServerBuildInfo get() {
```

Comment:
Are we sure about `get`? The other similar interface we added, `RegistryAccess`'s static getter method is `registryAccess()`. Am not the biggest fan of `get` due to static imports.